src/context/Auth/AuthContext.js

```diff
@@ -0,0 +1,34 @@
+import {createContext, useContext, useReducer, useEffect} from 'react';
+import { AuthReducer } from '../../Reducer/Auth/AuthReducer';
+
+const AuthContext = createContext();
+
+const initialState = {
+    user: "",
```

Comment:
if you will store an user data here , do give it an default value of Object `{}`